src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest.cs

```diff
@@ -222,7 +222,9 @@ public void BuildAotApplication (string supportedAbis, bool enableLLVM, bool exp
 			}
 			using (var b = CreateApkBuilder (path)) {
 				if (!b.CrossCompilerAvailable (supportedAbis))
-					Assert.Ignore ("Cross compiler was not available");
+					Assert.Ignore ($"Cross compiler for {supportedAbis} was not available");
```

Comment:
Perhaps I'm missing something, but if we `Assert.Ignore()` shouldn't we also immediately `return` so that we don't continue execution? (Or does `Assert.Ignore()` work by throwing an exception, thus exiting immediately? [The documentation](https://github.com/nunit/docs/wiki/Assert.Ignore) doesn't say.)